ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/pderesources.properties

```
@@ -2283,6 +2283,7 @@ GetNonExternalizedStringsAction_allExternalizedMessage=All strings in manifest f
 RequiredExecutionEnvironmentSection_title=Execution Environments
 RequiredExecutionEnvironmentSection_add=Add...
 RequiredExecutionEnvironmentSection_remove=Remove
+RequiredExecutionEnvironmentSection_change=Change
```

Like for the add-button we should also append three dots to indicate this opens a dialog.
```suggestion
RequiredExecutionEnvironmentSection_change=Change...
```